assets/js/script.js

```diff
@@ -96,50 +121,45 @@ $(document).ready(function() {
-                  <!-- Accordion tab title -->
-                  <a href=\"#\" class=\"accordion-title\">${response.meals[i].strMeal} </a>
-                </li>`
+            //   let meal = `<li class=\"accordion-item\" meal-id = ${response.meals[i].idMeal} data-accordion-item data-allow-all-closed='true' >
+            //       <!-- Accordion tab title -->
+            //       <a href=\"#\" class=\"accordion-title\">${response.meals[i].strMeal} </a>
+            //     </li>`
+
+                var accordianMealTest=       
+                 `<div class="grid-container" id="mainContent">
+                
+                    <ul class="accordion" data-accordion data-allow-all-closed="true" >
+                        <li class="accordion-item" data-accordion-item meal-id= ${response.meals[i].idMeal} >
+                        <!-- Accordion tab title -->
+                        <a href="#" class="accordion-title ">${response.meals[i].strMeal}</a>
+                        <div class="accordion-content panel" data-tab-content>
+                           
+                           
+                        </div>
+                        </li>
+                    </ul>
+            </div>`
+            //console.log(accordianMealTest);
             
-                console.log(meal);
+                //console.log(meal);
                 
                 // var callMeals = `<li class="mealClass" data-mealid="${response.meals[i].idMeal}"'>${response.meals[i].strMeal} </li> `;
                 // // (response.meals[i].strMeal);
                 // //console.log(callMeals);
                 // $('#searchResultsTab').append(callMeals);
-                $('#searchResultsTab').append(meal);
+                $('#searchResultsTab').append(accordianMealTest);
 
                 
                 
-                //$('#searchResults').append('<li>'+callMeals+'</li>',{class: 'mealClass'});
-              
-                
-                // $("#searchRsults ul").on("click", "li", function(){
-                //     searchMeals($(this).text());
-                //callback
-
-                // })
-                
                 
-                //$("#searchResults").append(callMeals);
-                // $('button').click(function() {
-                //     var mealList = $('#input').val();
-                //     $('#list').append('<li>'+mealList+'</li>');
-                //     //return false;
-                //  });
 
 
             }
-            //Adding class to 'li' in order to execute click function
-            //$('li').addclass('mealClass');
-            
-            // $('mealClass').class(onclick);
-            //$(selector).addClass(classname,function(index,currentclass))
-
-            //add here. class selector. assignt class to li.
+           
             //click listener
             registerClickListeners()
```

Comment:
The registerClickListeners() function is called but doesn't seem to be defined anymore  (deleted from line 52-69);
Is this function still required or will be redefined later?

Reply:
That bit was commented out. Sorry its a bit unclear. I still have that function. Thats my comments and previous code. So not affected. I was just deleting out all the comment madness.